challange_1.py

```diff
@@ -0,0 +1,6 @@
+a,b = 0,1
+
+for i in range(100000):
+    print(a%10)
+    a,b = b,a+b
+    
```

Comment:
How to evaluate 1 billionth Fibonacci number's first digit?

https://python.plainenglish.io/how-i-calculated-the-1-000-000th-fibonacci-number-with-python-e921d3642dbf

Reply:
```
for i in 1000000000:
    # evaluate
 ```
Upper is so inefficient. We can evaluate it by `log(1,000,000,000)` iterations.